sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/OpenTelemetrySdkAutoConfiguration.java

```diff
@@ -109,11 +110,14 @@ static OpenTelemetrySdk newOpenTelemetrySdk(
             spanExporterCustomizer,
             samplerCustomizer);
 
-    OpenTelemetrySdk openTelemetrySdk =
-        OpenTelemetrySdk.builder()
-            .setTracerProvider(tracerProvider)
-            .setPropagators(propagators)
-            .build();
+    OpenTelemetrySdkBuilder sdkBuilder =
+        OpenTelemetrySdk.builder().setTracerProvider(tracerProvider).setPropagators(propagators);
+
+    if (meterProvider instanceof SdkMeterProvider) {
+      sdkBuilder.setMeterProvider((SdkMeterProvider) meterProvider);
+    }
```

Comment:
We're stuck with this pattern of possibly not having an `SdkMeterProvider` until the metric SDK stabilizes and we remove the "metrics are off by default" feature, right?